src/components/header/header.tsx

```diff
@@ -128,30 +126,28 @@ const Header: React.FC<HeaderProps> = (props) => {
   });
 
   return (
-    <HeroImage image={props.heroImage}>
```

Comment:
Why was it structured like that? Hero nesting the header feels up side down.

Reply:
Don't ask :D